### DISCUSSION TITLE
Is it possible to add a Volume to the chart?

### DISCUSSION THREAD

**Discussion by timocov** [tradingview/lightweight-charts#633] [Q&A] [answered]
Is it possible to add a Volume to the chart and how to do that?

**Comment by timocov** [comment, marked as answer]
Yes, it is possible.

`lightweight-charts` knows nothing about what exactly you display in the chart, thus you can't show volume via adding `volume` field to the series' data. But you can create [Histogram](https://github.com/tradingview/lightweight-charts/blob/master/docs/histogram-series.md) series and put volume's data to it.

See example https://jsfiddle.net/TradingView/cnbamtuh/.

**Reply by haydenpang** [reply]
Is it possible to add a Net Volume (Buy Volume - Sell Volume)?

**Reply by timocov** [reply]
> Is it possible to add a Net Volume (Buy Volume - Sell Volume)?

@haydenpang What's the difference?

I mean, do you have any problem with adding this kind of indicator?

**Reply by julio899** [reply]
how I can add addLineSeries to HistogramSeries ? that's posible? @timocov  & thanks for all you helps
![image](https://user-images.githubusercontent.com/2575745/162803946-fa4fee99-5a52-468a-bf8b-9824ddb0a2f8.png)

**Reply by seohuisang** [reply]
Volume and bar charts overlap
